easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'skewer'
+version = '0.2.2'
+
+homepage = 'https://github.com/relipmoc/skewer'
+description = """skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter
```

Comment:
Well, I can change that if you really care, but the official software name is 'skewer', so...

I grabbed this from the README at https://github.com/relipmoc/skewer